osc-dm-product-srv/src/server.py

```diff
@@ -186,6 +186,7 @@ async def dataproducts_uuid_artifacts_get(uuid: str, artifact_uuid: str) -> mode
     """
     Discover product artifact by uuid
     """
+    logger.info(f"***CALLED ENDPOINT***")
```

Comment:
should include information on what url was called. something like "called endpoint /uuid/<uuid>/artifact/<artifact>" so that this log message is actually useful to anyone reading the logs.

Reply:
was left in from testing. log statement removed